AdobeStockAsset/Test/Integration/Model/SaveAssetTest.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockAsset\Test\Integration\Model;
+
+use Magento\AdobeStockAssetApi\Api\AssetRepositoryInterface;
+use Magento\AdobeStockAssetApi\Api\CategoryRepositoryInterface;
+use Magento\AdobeStockAssetApi\Api\CreatorRepositoryInterface;
+use Magento\AdobeStockAssetApi\Api\Data\AssetInterface;
+use Magento\AdobeStockAssetApi\Api\Data\AssetInterfaceFactory;
+use Magento\AdobeStockAssetApi\Api\Data\CategoryInterface;
+use Magento\AdobeStockAssetApi\Api\Data\CategoryInterfaceFactory;
+use Magento\AdobeStockAssetApi\Api\Data\CreatorInterface;
+use Magento\AdobeStockAssetApi\Api\Data\CreatorInterfaceFactory;
+use Magento\AdobeStockAssetApi\Api\SaveAssetInterface;
+use Magento\MediaGalleryApi\Api\Data\AssetInterface as MediaAsset;
+use Magento\MediaGalleryApi\Api\Data\AssetInterfaceFactory as MediaAssetFactory;
+use Magento\MediaGalleryApi\Model\Asset\Command\DeleteByPathInterface;
+use Magento\MediaGalleryApi\Model\Asset\Command\SaveInterface;
```

Comment:
@coderimus can you please update the MediaGalleryApi interfaces used in this PR to the new bulk interfaces